tests/interactive_broker_version.py

```diff
@@ -33,229 +25,16 @@ def version_as_number(version):
 
 def test_version(version, cmd=None, deploy=True, conf={}, debug=False,
                  exec_cnt=1,
-                 root_path='tmp', broker_cnt=3, scenario='default'):
+                 root_path='tmp', broker_cnt=3, scenario='default',
+                 kraft=False):
     """
     @brief Create, deploy and start a Kafka cluster using Kafka \\p version
-    Then run librdkafka's regression tests.
+    Then run librdkafka's regression tests. Use inherited environment.
     """
-
-    print('## Test version %s' % version)
-
-    cluster = Cluster('LibrdkafkaTestCluster', root_path, debug=debug)
-
-    if conf.get('sasl_oauthbearer_method') == 'OIDC':
-        oidc = OauthbearerOIDCApp(cluster)
-
```

Comment:
I think we should move whole deleted part to test_it instead of removing them. If we use other version in future then we should have fail fast working fine. Same for the other file as well.

Reply:
The `OauthbearerOIDCApp `instance is now created by `LibrdkafkaTestCluster` in `test_it`

Reply:
I am talking about the whole deleted part. Are we testing them already in the test_it function?

Reply:
this part is replaced by the `LibrdkafkaTestCluster` in `test_it` the code was very similar